eole/predict/inference.py

```diff
@@ -650,17 +653,17 @@ def _decode_and_generate(
         self,
         decoder_in,
         enc_out,
-        batch,
         src_len,
         step=None,
-        batch_offset=None,
         return_attn=False,
     ):
 
         # Decoder forward, takes [batch, tgt_len, nfeats] as input
         # and [batch, src_len, hidden] as enc_out
         # in case of inference tgt_len = 1, batch = beam times batch_size
         # in case of Gold Scoring tgt_len = actual length, batch = 1 batch
+        emb = self.model.tgt_emb(decoder_in, step=step)
```

Comment:
Specific reason for moving this?

Reply:
not anymore, it's just that as an intermediary change I tried to get rid of sequence_mask() but the tiling thing prevented me to do so.